tests/components/directv/__init__.py

```diff
@@ -46,7 +43,7 @@ def mock_connection(aioclient_mock: AiohttpClientMocker) -> None:
     aioclient_mock.get(
         f"http://{HOST}:8080/info/mode",
         params={"clientAddr": "9XXXXXXXXXX9"},
-        status=HTTP_INTERNAL_SERVER_ERROR,
+        status=HTTPStatus.INTERNAL_SERVER_ERROR.value,
```

Comment:
Or is there a reason to still use `.value` here ?

Reply:
Nope, I messed up something with this PR yesterday, I actually removed the `.value`s but dunno what happened. Should be cleaned up now.